datalad_registry/search.py

```diff
@@ -53,7 +53,7 @@ def get_branches_ilike_search(value):
 known_fields = {
     _: partial(get_ilike_search, RepoUrl, _) for _ in known_fields_RepoUrl_1to1
 }
-known_fields["metadata"] = get_metadata_ilike_search  # type: ignore
+
```

Comment:
This removes search within 'metadata' field entirely, which definitely would cause failures

Reply:
Yeah, I just found that out as well. I have fixed that, so all tests passe at this point.